doc/mkdocs.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+site_name: 'Exploring Pravega-operator'
+site_url: 'http://pravega.io/'
+site_description: 'Documentation to help you get familiar with Pravega-operator'
+
+repo_url: 'https://github.com/pravega/pravega'
+edit_uri: 'tree/master/doc'
```

Comment:
Should be 'tree/master/doc/docs'

Reply:
@Tristan1900 Accepted suggestion

Reply:
@Tristan1900 but a folder named docs should be created and we need to move all `.md` files into it...Please check...

Reply:
Yes, that is right. It seems we need to move all `.md` into the `/docs` directory according to [this](https://www.mkdocs.org/about/release-notes/#stricter-directory-validation), it doesn't allow `docs_dir` to have config file(`mkdocs.yaml`) now.